parameters.yaml

```diff
@@ -6,3 +6,12 @@ parameters:
     required: true
     type: string
     description: Cluster ID
+
+  OrganizationIdPathParameter:
+    name: organization_id
+    in: path
+    required: true
+    type: string
+    description: |
+        An ID for the organization to be created is the only required parameter.
+        This ID can only contain alpha-numeric characters, hyphens, underscores, and dots.
```

Comment:
The actual requirements are more strict. It has to match `[a-z0-9_]{4,30}`, so no hyphens, dots, And it has to be unique.

See https://github.com/giantswarm/giantswarm/issues/15